src/FormattedMessage.jsx

```diff
@@ -23,8 +25,16 @@ const FormattedMessage = (props) => (
       const style = {
         fontSize: 'inherit',
       };
+
       /* eslint-disable react/no-danger */
-      return <span style={style} dangerouslySetInnerHTML={{ __html: messageString }} />;
+      return (
+        <span
+          automation-id={id}
```

Comment:
Need the same ability for automation-id as well, if users are specifying an automation id explicitly, then we need to use that.

Reply:
@MadaraUchiha-314 Updated to allow custom automation id. Let me know if it looks good, thanks.